pallets/keystore/src/lib.rs

```diff
@@ -1,15 +1,30 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
+#[cfg(test)]
+mod mock;
+
+#[cfg(test)]
+mod tests;
+
+#[cfg(feature = "runtime-benchmarks")]
+mod benchmarking;
+
+mod weights;
+
 pub use pallet::*;
+//use weights::*;
```

Comment:
dead code?

Reply:
Iirc we're going to need that once benchmarking is implemented

Reply:
Yes, that was the intent.